src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -2142,11 +2142,15 @@ public void action() {
 
     private class PrintPreviewAction implements BaseAction {
 
-         @Override
-         public void action() throws Throwable {
+        @Override
+        public void action() throws Throwable {
```

Comment:
Throwing a throwable is usually a bad idea:
http://stackoverflow.com/questions/12564428/is-throws-throwable-good-practice
Change throwable to Exception, if no more specific exception is avaiable.